rmgpy/molecule/group.py

```diff
@@ -1375,10 +1392,13 @@ def clear_reg_dims(self):
             atm.reg_dim_atm = [[], []]
             atm.reg_dim_u = [[], []]
             atm.reg_dim_r = [[], []]
+            atm.reg_dim_site = [[],[]]
+            atm.reg_dim_morphology = [[],[]]
         for bd in self.get_all_edges():
             bd.reg_dim = [[], []]
 
-    def get_extensions(self, r=None, basename='', atm_ind=None, atm_ind2=None, n_splits=None):
+    def get_extensions(self, r=None, r_bonds=[1,2,3,1.5,4], r_un=[0,1,2,3], basename='', atm_ind=None, atm_ind2=None, n_splits=None):
```

Comment:
Can you set the defaults to `None` and set them within the function? It's bad practice to have mutable defaults.

Reply:
Can you add a test for this function too?

Reply:
Changed. This function has a test already.